pkg/pgmodel/end_to_end_tests/promql_endpoint_integration_test.go

```diff
@@ -122,30 +126,35 @@ func testRequest(req *http.Request, handler http.Handler, client *http.Client, c
 	}
 }
 
-func testRequestConcurrent(requestCases []requestCase, handler http.Handler, client *http.Client, comparator resultComparator) func(*testing.T) {
+func testRequestConcurrent(requestCases []requestCase, client *http.Client, comparator resultComparator) func(*testing.T) {
 	return func(t *testing.T) {
 
 		perm := rand.Perm(len(requestCases))
 
 		wg := sync.WaitGroup{}
 
 		for i := 0; i < len(perm); i++ {
-			req := requestCases[perm[i]].req
+			tsReq := requestCases[perm[i]].tsReq
+			promReq := requestCases[perm[i]].promReq
 			log := requestCases[perm[i]].log
 			wg.Add(1)
-			go func() {
+			go func(log string) {
```

Comment:
why was this change necessary? Log is created inside the loop so it should be local to the closure

Reply:
I'll remove that, its leftover, not used anymore.